nimble/controller/include/controller/ble_ll_scan.h

```diff
@@ -88,14 +88,15 @@ struct ble_ll_scan_params
     uint32_t next_event_start;
 };
 
-#define BLE_LL_AUX_CHAIN_BIT            0x01
-#define BLE_LL_AUX_INCOMPLETE_BIT       0x02
-#define BLE_LL_AUX_INCOMPLETE_ERR_BIT   0x04
-#define BLE_LL_AUX_HAS_ADDRA            0x08
-#define BLE_LL_AUX_IGNORE_BIT           0x10
-#define BLE_LL_AUX_HAS_DIR_ADDRA        0x20
-#define BLE_LL_AUX_TRUNCATED_SENT       0x40
-#define BLE_LL_AUX_HAS_ADI              0x80
+#define BLE_LL_AUX_CHAIN_BIT            0x0001
+#define BLE_LL_AUX_INCOMPLETE_BIT       0x0002
+#define BLE_LL_AUX_INCOMPLETE_ERR_BIT   0x0004
+#define BLE_LL_AUX_HAS_ADDRA            0x0008
+#define BLE_LL_AUX_IGNORE_BIT           0x0010
+#define BLE_LL_AUX_HAS_DIR_ADDRA        0x0020
+#define BLE_LL_AUX_TRUNCATED_SENT       0x0040
+#define BLE_LL_AUX_HAS_ADI              0x0080
+#define BLE_LL_SENT_EVENT_TO_HOST       0x0100
```

Comment:
I'm not sure if I get it why this flag is needed,  some code comment would be nice :)